### PR TITLE
fix(popup): show SiteNotMonetized state for non-monetized websites

### PR DESCRIPTION
**Pull request by sidvishnoi**
## Context

Closes https://github.com/interledger/web-monetization-extension/issues/318

## Changes proposed in this pull request

- Replaces payment form with an error message/empty state if site isn't monetized.
- Moves logic to get popup data and check if site is monetized to `MonetizationService`. Use Storage service only to return stored data.

---

- [x] TODO: UI Design and copy-edit.
     - https://github.com/interledger/web-monetization-extension/issues/318#issuecomment-2176514411
- [x] update UI when popup is open and then all tags are removed 
- [x] Question: Do we want to show the payment slider? Or show an entirely different screen? I think entirely new screen.
   - Decided to show entire screen.

### CODE REVIEW

**Comment by raducristianpopa** [comment]
<h2>Extension builds preview</h2>

<table role="table">
<thead>
  <tr>
    <th>Name</th>
    <th>Link</th>
  </tr>
</thead>
<tbody>
  <tr>
    <td align="center">Latest commit</td>
    <td align="center">c3735904db084b1471e79b9462a9f885a6965dc4</td>
  </tr>
  <tr>
    <td align="center">Latest job logs</td>
    <td align="center"><a href="https://github.com/interledger/web-monetization-extension/actions/runs/9548702020">Run #9548702020</a></td>
  </tr>
  <tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/24988772422/artifacts/1608650919">Download</a></td></tr><tr><td align="center">Badge</td><td align="center"><a href="https://github.com/interledger/web-monetization-extension/suites/24988772422/artifacts/1608650921">Download</a></td></tr>
</tbody>
</table>

**Comment by sidvishnoi** [comment]
Current preview:
image

**Review by dianafulga — APPROVED** [review]

**Comment by tselit** [comment]
We discussed and proposed the following simplified information UI:
image

**Comment by tselit** [comment]
Just updating this PR about the UI that was agreed on here 👇🏾 
https://github.com/interledger/web-monetization-extension/issues/318#issuecomment-2176514411

**Comment by sidvishnoi** [comment]
Updated UI:
image

File: src/popup/components/ProtectedRoute.tsx [hunk 1]
```diff
@@ -12,6 +13,9 @@ export const ProtectedRoute = () => {
   if (state.connected === false) {
     return <Navigate to={ROUTES_PATH.SETTINGS} />
   }
+  if (!state.isSiteMonetized) {
+    return <SiteNotMonetized />
+  }
```
[the remark this answers is not in the document]
Comment (sidvishnoi) [reply src/popup/components/ProtectedRoute.tsx:18]: Good idea! I was thinking of keeping such logic out of Home for some reason.

**Review by raducristianpopa — APPROVED** [review]

**Pull request merged**